metis-enrichment/metis-enrichment-service/src/main/java/eu/europeana/enrichment/service/Converter.java

```diff
@@ -1,74 +1,62 @@
 package eu.europeana.enrichment.service;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
+import eu.europeana.corelib.solr.entity.AbstractEdmEntityImpl;
 import eu.europeana.corelib.solr.entity.AgentImpl;
 import eu.europeana.corelib.solr.entity.ConceptImpl;
 import eu.europeana.corelib.solr.entity.PlaceImpl;
 import eu.europeana.corelib.solr.entity.TimespanImpl;
-import eu.europeana.enrichment.api.external.EntityWrapper;
 import eu.europeana.enrichment.api.external.model.Agent;
 import eu.europeana.enrichment.api.external.model.Concept;
 import eu.europeana.enrichment.api.external.model.EnrichmentBase;
-import eu.europeana.enrichment.api.external.model.EnrichmentBaseWrapper;
-import eu.europeana.enrichment.api.external.model.EnrichmentResultList;
 import eu.europeana.enrichment.api.external.model.Label;
 import eu.europeana.enrichment.api.external.model.LabelResource;
 import eu.europeana.enrichment.api.external.model.Part;
 import eu.europeana.enrichment.api.external.model.Place;
 import eu.europeana.enrichment.api.external.model.Resource;
 import eu.europeana.enrichment.api.external.model.Timespan;
-import java.io.IOException;
+import eu.europeana.enrichment.api.internal.MongoTermList;
+import eu.europeana.enrichment.utils.EntityType;
+import eu.europeana.enrichment.utils.EntityTypeUtils;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
-import org.springframework.stereotype.Service;
 
 /**
  * Contains functionality for converting from an incoming Object to a different one.
  */
-@Service
-public class Converter {
-
-  /**
-   * Converts a {@link List<EntityWrapper>} to an {@link EnrichmentResultList}
-   *
-   * @param wrapperList the {@link List<EntityWrapper>} to convert
-   * @return the result of the convertion
-   * @throws IOException if conversion fails
-   */
-  public EnrichmentResultList convert(List<EntityWrapper> wrapperList) throws IOException {
-    EnrichmentResultList list = new EnrichmentResultList();
-
-    for (EntityWrapper wrapper : wrapperList) {
-      list.getEnrichmentBaseWrapperList().add(new EnrichmentBaseWrapper(wrapper.getOriginalField(), convert(wrapper)));
+public abstract class Converter {
```

Comment:
My IDE is giving me some warnings here. Perhaps:
- Make final instead of abstract (because sonarlint complains that an abstract class does not have instances).
- Add private no-args constructor

Reply:
Adding IDE suggestion